lib/posix/options/Kconfig.file_system_r

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config POSIX_FILE_SYSTEM_R
+	bool "Thread-Safe File System"
+	select FILE_SYSTEM
+	select FDTABLE
+	help
+	  Select 'y' here and Zephyr will provide an implementation of the POSIX_FILE_SYSTEM_R
+	  Option Group, consisting of readdir_r().
+
+	  For more informnation, please see
```

```suggestion
	  For more information, please see
```

On my phone but I'm guessing there might be other occurrences elsewhere, so fixing as a follow-up is fine